Paws4Me/Paws4Me/Extensions/UIImageExtension.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  UIImageExtension.swift
+//  Paws4Me
+//
+//  Created by Kirsty-Lee Walker on 2022/03/09.
+//
+
+import Foundation
+import UIKit
+
+extension UIImage {
+    public static func loadFrom(url: URL, completion: @escaping (_ image: UIImage?) -> Void) {
+        DispatchQueue.global().async {
+            if let data = try? Data(contentsOf: url) {
+                DispatchQueue.main.async {
+                    completion(UIImage(data: data))
+                }
+            } else {
+                DispatchQueue.main.async {
+                    completion(nil)
+                }
+            }
+        }
+    }
+    public static func displayImgFromUrl(url: String, petImageView: UIImageView)
+    -> UIImageView {
+        guard let url = URL(string: url) else { return UIImageView() }
+        UIImage.loadFrom(url: url) {image in
+            petImageView.layer.cornerRadius = 10
```

Comment:
So code like this belongs in an extension separately because this function is doing too much 😄 if you ever wanted to download another image that you didn't want to round then this would be an issue.